### PR TITLE
readme-update

### PR DESCRIPTION
**Pull request by jpvajda**
This PR updates our readme. 📖

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,49 @@
...
 
-Contribute your own observability pack to the New Relic One catalog by following the steps below:
+Contribute your own quick start to the New Relic One catalog by following the steps below:
 
 1. [Fork the Github repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository)
 
 2. [Clone your own repository to your local machine](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
 
-3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your pack, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`
+3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your Quick start, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`
```
Comment (rudouglas) [inline README.md:42]: just changing to small case to keep it consistent
```suggestion
3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your quick start, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`
```

File: README.md [hunk 2]
```diff
@@ -1,30 +1,49 @@
...
 
-4. In your new directory, you'll find the following folders: `dashboards`, `alerts`, `instrumentation`. Each folder contains a template or template directories that you can use to create entities for your observability pack.
+4. In your new directory, you'll find the following folders: `dashboards`, `alerts`, `instrumentation`. Each folder contains a template or template directories that you can use to create entities for your Quick start.
```
Comment (rudouglas) [inline README.md:44]: Case again, also do we want to reference `instrumentation` yet when it's not quite a thing? 🤷 
```suggestion
4. In your new directory, you'll find the following folders: `dashboards`, `alerts`, `instrumentation`. Each folder contains a template or template directories that you can use to create entities for your quick start.
```

File: README.md [hunk 3]
```diff
@@ -35,7 +54,7 @@ Contribute your own observability pack to the New Relic One catalog by following
 
     This process is similar for all other entity directories. Also, if you don't want to create entities for a given type, delete the corresponding directory.
 
-5. In your pack's root directory, you'll find a `config.yaml` file. Set the name of your pack. Everything else is optional.
+5. In your pack's root directory, you'll find a `config.yml` file where you can configure your Quick start. Refer to our Contributing Guide(./CONTRIBUTING.md) for more details on Quick Start configurations.
```
Comment (rudouglas) [inline README.md:57]: casing
```suggestion
5. In your pack's root directory, you'll find a `config.yml` file where you can configure your quick start. Refer to our Contributing Guide(./CONTRIBUTING.md) for more details on quick start configurations.
```

**Review by rudouglas — COMMENTED** [review]
LG just a few case changes to keep `quick starts` consistent

**Review by rudouglas — CHANGES_REQUESTED** [review]
couple of case changes needed

**Comment by rudouglas** [comment]
Oh lets 🅿️ this but i think it should be `quickstarts` rather than `quick starts`

File: README.md (hunk 1 above)
Comment (aswanson-nr) [reply README.md:42]:
```suggestion
3. Copy the `_template` directory and its content to a new directory within the `packs` folder. Choose a name which identifies the purpose of your quickstart, such as `rabbitmq`, `apm-errors`, `sre`, or `aws-s3`
```

File: README.md (hunk 2 above)
Comment (aswanson-nr) [reply README.md:44]:
```suggestion
4. In your new directory, you'll find the following folders: `dashboards`, `alerts`, `instrumentation`. Each folder contains a template or template directories that you can use to create entities for your quickstart.
```

File: README.md (hunk 3 above)
Comment (aswanson-nr) [reply README.md:57]:
```suggestion
5. In your pack's root directory, you'll find a `config.yml` file where you can configure your quickstart. Refer to our Contributing Guide(./CONTRIBUTING.md) for more details on quickstart configurations.
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Found a few more spots to update!

**Comment by jpvajda** [comment]
@aswanson-nr I fixed all those... I'll never forget this syntax again 🤣

File: README.md [hunk 4]
```diff
@@ -35,7 +54,7 @@ Contribute your own observability pack to the New Relic One catalog by following
 
     This process is similar for all other entity directories. Also, if you don't want to create entities for a given type, delete the corresponding directory.
 
-5. In your pack's root directory, you'll find a `config.yaml` file. Set the name of your pack. Everything else is optional.
+5. In your pack's root directory, you'll find a `config.yml` file where you can configure your quickstart. Refer to our Contributing Guide(./CONTRIBUTING.md) for more details on Quick Start configurations.
```
Comment (aswanson-nr) [inline README.md:57]:
```suggestion
5. In your pack's root directory, you'll find a `config.yml` file where you can configure your quickstart. Refer to our Contributing Guide(./CONTRIBUTING.md) for more details on quickstart configurations.
```

**Review by aswanson-nr — APPROVED** [review]

**Comment by jpvajda** [comment]
@rudouglas has to approve since he had a changed requested as well. 🙏

**Pull request merged**